fuel/app/classes/controller/welcome.php

```diff
@@ -16,42 +17,38 @@
  * A basic controller example.  Has examples of how to set the
  * response body and status.
  *
- * @package  app
  * @extends  Controller
  */
-class Controller_Welcome extends Controller
+class controller_welcome extends Controller
```

Comment:
__[code_sniffer]__
* Class name must begin with a capital letter
* Class name is not valid; consider Controller_Welcome instead